tests/base.py

```diff
@@ -160,7 +161,8 @@ def expected_metadata(self):
             },
             "roles": {
                 self.PRIMARY_KEYS: {'id'},
-                self.REPLICATION_METHOD: self.FULL
+                self.REPLICATION_METHOD: self.FULL,
+                self.START_DATE_KEY: 'updated_at'
```

Comment:
Why can't we use created_at for roles or employees?

Reply:
updated_at is always preferred, why would we want it to be created_at? What if a record older than the created_at date is updated?

Reply:
That makes sense